control_panel_api/auth0.py

```diff
@@ -1,3 +1,5 @@
+from collections import OrderedDict
```

Comment:
Why `users_to_add` needs to be an ordered dictionary? As far as I can tell order shouldn't matter. What am I missing?

Reply:
As it says in the comment on L62, the dict is sorted to make testing easier.